pkg/services/user/userimpl/store.go

```diff
@@ -403,8 +403,10 @@ func (ss *sqlStore) GetSignedInUser(ctx context.Context, query *user.GetSignedIn
 		org.name              as org_name,
 		org_user.role         as org_role,
 		org.id                as org_id,
-		u.is_service_account  as is_service_account
+		u.is_service_account  as is_service_account,
+		user_auth.auth_module as authenticated_by
 		FROM ` + ss.dialect.Quote("user") + ` as u
+		LEFT OUTER JOIN user_auth on user_auth.user_id = u.id
```

Comment:
A user can have multiple user_auth entries, in my test sqlite this query was returning the oldest user_auth as the first result. 

This might send information that doesn't match the retrieval from the AuthInfo service